### DISCUSSION TITLE
Map Spotify to eject key

### DISCUSSION THREAD

**Discussion by occasionalfindings** [pqrs-org/Karabiner-Elements#3919] [Q&A]
Hi there,

I am looking for a way to map the Spotify app to the eject key on the Apple Magic Keyboard. Controlling Spotify is done through [MacMediaKeyFowarder](https://github.com/milgra/macmediakeyforwarder), but I haven't found a way to start Spotify with a keypress.

Any help would be greatly appreciated.

Cheers :)

**Comment by raxigan** [comment]
An example rule for opening Spotify with ctrl + opt + S. You can adjust it to your needs.
```json
{
  "description": "Ctrl + Opt + S (Open Spotify)",
  "manipulators": [
    {
      "from": {
        "key_code": "s",
        "modifiers": {
          "mandatory": [
            "control",
            "option"
          ],
          "optional": [
          ]
        }
      },
      "to": [
        {
          "shell_command": "open -a Spotify"
        }
      ],
      "type": "basic"
    }
  ]
}
```

**Comment by occasionalfindings** [comment]
Thanks for your quick reply @raxigan .

I am at a loss how to add this as a complex formula. The screenshot shows an example for Eject to lock, but I fail to add your script without receiving any errors.

![grafik](https://github.com/user-attachments/assets/c9bb2bb1-e566-4c2a-b82b-10e9ebce0d8b)

**Comment by raxigan** [comment]
Yep, `title` and `rules` should be omitted if you paste it in UI. I edited my post.
